src/lib/journey/callbacks/confirmation/confirmation.svelte

```diff
@@ -97,43 +97,38 @@
   }
 </script>
 
-<!-- Only render confirmation if NOT currently on 'OneTimePassword' stage -->
-{#if stepMetadata?.platform?.stageName !== 'OneTimePassword'}
-  {#if !stepMetadata?.derived.isStepSelfSubmittable()}
-    {#if options.length > 1}
-      <Select
-        isFirstInvalidInput={callbackMetadata?.derived.isFirstInvalidInput || false}
-        isRequired={false}
-        key={inputName}
-        {label}
-        onChange={setOptionValue}
-        {options}
-      />
-    {:else}
-      <Checkbox
-        isFirstInvalidInput={callbackMetadata?.derived.isFirstInvalidInput || false}
-        isInvalid={false}
-        key={inputName}
-        onChange={setCheckboxValue}
-        value={false}
-      >
-        {options[0].text}
-      </Checkbox>
-    {/if}
+{#if !stepMetadata?.derived.isStepSelfSubmittable()}
```

Comment:
I'm removing the outer conditional here to reduce coupling between stages and callbacks. The `ConfirmationCallback` is now being removed from the callback array in these custom stages, so this no longer gets picked up in the callback iteration, but used separately in the custom stage.